spec/features/user_add_review_to_item_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "rails_helper"
+
+feature "user creates a review for an item" do
+  before(:each) do
+    user = FactoryGirl.create(:user)
+    visit root_path
+    click_link 'Sign In'
+
+    fill_in 'Electronic Mail', with: user.email
+    fill_in 'Password', with: user.password
+    click_button 'Sign In'
+
+    visit new_item_path
+    fill_in "Title", with: "Pokemon"
+    fill_in "Description", with: "Gotta Catch 'Em All"
+    click_button "Create Item"
+
+    visit items_path
+    click_link 'Pokemon'
+  end
+
+  scenario "users see reviews on show page once they are made" do
+    click_link 'Add Review'
+
+    fill_in "Rating", with: 3
+    fill_in "Body", with: "Review the first"
+    click_button 'Create Review'
+
+    expect(page).to have_content "Pokemon"
+    expect(page).to have_content 3
+    expect(page).to have_content "Review the first"
+  end
+
+  scenario "users are able to delete reviews they have created" do
+    
```

Comment:
No biggie, but in the future, please delete code you're not using/that's not relevant to the particular PR!